### ISSUE TITLE
Shift Click when Out of Raid Opens Loot Window

### ISSUE THREAD

**Issue by Gogo1951** [papa-smurf/Gargul#36]
Maybe this is how you hand loot out? I don't know, never saw this in the raid but when I shift click on an item at the AH it opens a loot window. 

![image](https://user-images.githubusercontent.com/55365231/163267208-ac1cfe4e-b47b-4a6e-ab0f-a4b692b2ad0b.png)

It shouldn't open when I'm not in a raid. Just odd behavior. Shift Click at the AH already has functionality bound to it.

I think... let me play around with this a bit more, knowing now how to open the loot window. 

Overall first impression is that there's way too much text. Nobody reads. Can't rely on instructions to make things work, it's gotta dummy proof intuitive. (=

https://www.amazon.com/Dont-Make-Think-Revisited-Usability/dp/0321965515

**Comment by papa-smurf** [comment]
You gotta make sure your comments are dummy proof as well because I don't understand what text you're referring to based on the screenshot? :)

Also: this window opens on alt+shift+click unless you changed the hotkeys (/gl > shortcut keys)

**Comment by Gogo1951** [comment]
When I'm at the AH, and I shift Click an item, not alt-shift-click, the Gargul window was opening. I wasn't in a raid, I wasn't ML. Just this came up when I tried to shift click on names of items so I could search for them in TSM Auction Window.

**Comment by papa-smurf** [comment]
Can you share your hotkey settings?

**Comment by Gogo1951** [comment]
Whatever Gargul came with by default is what I've got.

I think a check if I'm in a raid or master looter would solve the problem, regardless of what toon or what keybinds I have.

Other than action bars, all my keybinds are default.

Ctrl + Shift + Z reports dmg via Details.

Ctrl + Z / X / C changes gear sets via Outfitter.

Alt + 1 / 2 moves camera zoom.

Past that I don't think I've configured anything special.
